forc-index-rs/sample-project/Forc.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+authors = ["Z"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "counter-contract"
+
+[project.metadata.indexing]
+namespace = "counter-contract"
+identifier = "counter-contract"
```

Comment:
looks a bit strange using 2 variables here with `counter-contract` as the value. Can we have unique values for each one?

Reply:
this is being ignored; we dont validate this so doesnt really matter what they are. Can remove one of these if we want.